components/navigation.tsx

```diff
@@ -18,6 +18,7 @@ const navItems = [
 export default function Navigation() {
   const [isOpen, setIsOpen] = useState(false)
   const [isScrolled, setIsScrolled] = useState(false)
+  const [loginOpen, setLoginOpen] = useState(false)
```

Comment:
A good practice is storing  boolean values as a question, use isLoginOpen/setIsLoginOpen instead.